crates/astria-composer/src/executor/mod.rs

```diff
@@ -226,10 +233,14 @@ impl Executor {
 
         let reset_time = || Instant::now() + self.block_time;
 
-        loop {
+        let reason = loop {
             select! {
                 biased;
 
+                () = self.shutdown_token.cancelled() => {
+                    info!("shutdown signal received");
```

Comment:
That info is redundant. You are going to print the same info twice immediately after exiting this loop